panoptes_client/avatar.py

```diff
@@ -0,0 +1,31 @@
+from panoptes_client.panoptes import (
+    Panoptes,
+    PanoptesObject,
+    LinkResolver,
+)
+from panoptes_client.project import Project
+
+
+class Avatar(PanoptesObject):
+    _api_slug = 'avatar'
+    _link_slug = 'avatars'
+    _edit_attributes = ()
+
+    @classmethod
+    def http_get(cls, path, params={}, headers={}):
+        project = params.pop('project')
```

Comment:
What's the intention here? I'm not sure `http_get` is the right place to do this. (This breaks `Avatar.find(1234)` for example.) Overriding the `.where()` method might be what you want. See `Classification` for [an example](https://github.com/zooniverse/panoptes-python-client/blob/9349cfa350112fdb59fa8d6dbad138e00a94002b/panoptes_client/classification.py#L12).

Reply:
The motivation behind http_get was because what I needed specifically was the url for a project's avatar via `project.avatar['media'][0]['src']`. However the http_get method does return more than just the url. I've nenamed it for now to 'get_avatar' to better reflect what it is doing.

Reply:
Ah, I see! Yes, that's a better name as there's a `http_get` method in the parent `PanoptesObject` class that this was overriding.